### ISSUE TITLE
Can we add watermark?

### ISSUE THREAD

**Issue by anarsamadov** [ctessier/nova-advanced-image-field#88]
Thanks for such a great package.

Can we use watermark api as in "Intervention Image"?

Example:
$img->insert('public/watermark.png');

**Comment by Ruitjes** [comment]
@anarsamadov Yes, I think it's quite easy to implement this for yourself and create a PR so we all contribute to this library.
Check: https://image.intervention.io/v2/api/insert
